pinot-core/src/main/java/org/apache/pinot/core/data/manager/SegmentDataManager.java

```diff
@@ -29,7 +29,7 @@
   private int _referenceCount = 1;
 
   @VisibleForTesting
-  synchronized int getReferenceCount() {
+  public synchronized int getReferenceCount() {
```

Comment:
I hope this API is still only for testing. The API does not make sense in production since the ref count could change any time after you get it. Please confirm.

Reply:
Yes, we are using for testing only, to assert that the DimensionTable correctly releases segments after loading.